tests/integration.rs

```diff
@@ -2222,6 +2233,55 @@ mod tests {
         }
 
         #[test]
+        #[cfg(target_arch = "aarch64")]
+        fn test_aarch64_pe_boot() {
+            test_block!(tb, "", {
+                let mut bionic = UbuntuDiskConfig::new(BIONIC_IMAGE_NAME.to_string());
+                let mut focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+
+                vec![
+                    &mut bionic as &mut dyn DiskConfig,
+                    &mut focal as &mut dyn DiskConfig,
+                ]
+                .iter_mut()
+                .for_each(|disk_config| {
+                    let guest = Guest::new(*disk_config);
+
+                    let mut workload_path = dirs::home_dir().unwrap();
+                    workload_path.push("workloads");
+
+                    let mut kernel_path = workload_path;
+                    kernel_path.push("Image");
+
+                    let mut child = GuestCommand::new(&guest)
+                        .args(&["--cpus", "boot=1"])
+                        .args(&["--memory", "size=512M"])
+                        .args(&["--kernel", kernel_path.to_str().unwrap()])
+                        .default_disks()
+                        .default_net()
+                        .args(&["--cmdline", ARM64_KERNEL_CMDLINE])
+                        .args(&["--seccomp", "false"])
+                        .spawn()
+                        .unwrap();
+
+                    thread::sleep(std::time::Duration::new(20, 0));
+
+                    aver_eq!(tb, guest.get_cpu_count().unwrap_or_default(), 1);
+                    // In AArch64 VM, experimental results show that when a new VM is booted,
+                    // the total available guest memory will be slightly smaller than that
+                    // in x86_64. Therefore here we slightly decrease the number.
+                    aver!(tb, guest.get_total_memory().unwrap_or_default() > 490_000);
```

Comment:
You can remove the comment above and update this value to `480_000` as this is the new value we applied everywhere in the CI for x86_64.

Reply:
Done. Thanks @sboeuf !